src/cloudant/_common_util.py

```diff
@@ -268,16 +268,24 @@ def get_docs(r_session, url, encoder=None, headers=None, **params):
 #pylint: disable=unused-argument
 def append_response_error_content(response, **kwargs):
     """
-    Provides a helper to add HTTP response error and reason messages.
+    Provides a helper to act as callback function for the response event hook
+    and add a HTTP response error with reason message to `response.reason`.
+    The `response` and `**kwargs` are necessary for this function to
+    properly operate as the callback.
+
+    :param response HTTP response object
+    :param kwargs HTTP request parameters
```

Comment:
Again, probably does not matter right now but to be thorough, the syntax here should be `:param response:` and `:param kwargs:`